simple_diarizer/cluster.py

```diff
@@ -5,7 +5,7 @@
 from scipy.ndimage import gaussian_filter
 from sklearn.cluster import AgglomerativeClustering, KMeans, SpectralClustering
 from sklearn.metrics import pairwise_distances
-
+from .Spectral_clustering import NME_SpectralClustering
```

Comment:
To be "pythonic", use lowercase letter for "spectral_clustering" (and rename the py file accordingly)

Reply:
Done